pyotgw/pyotgw.py

```diff
@@ -383,30 +416,30 @@ def get_mode(self):
         """
         if not self._connected:
             return
-        return self._protocol.status.get(OTGW_MODE)
+        return self._protocol.status.get(v.OTGW_MODE)
 
-    async def set_mode(self, mode, timeout=OTGW_DEFAULT_TIMEOUT):
+    async def set_mode(self, mode, timeout=v.OTGW_DEFAULT_TIMEOUT):
         """
         Set the operating mode to either "Gateway" mode (:mode: =
-        OTGW_MODE_GATEWAY or 1) or "Monitor" mode (:mode: =
-        OTGW_MODE_MONITOR or 0), or use this method to reset the device
-        (:mode: = OTGW_MODE_RESET).
+        v.OTGW_MODE_GATEWAY or 1) or "Monitor" mode (:mode: =
+        v.OTGW_MODE_MONITOR or 0), or use this method to reset the device
+        (:mode: = v.OTGW_MODE_RESET).
```

Comment:
`v.` should be kept out of docstrings. It has no meaning outside this file.

Reply:
Once this is addressed we can merge.

Reply:
Done! 👍